test-page/src/views/Category.vue

```diff
@@ -125,7 +125,7 @@
             "
             :regular-price="product.calculatedPrice.unitPrice"
             :isOnWishlist="false"
-            @click:wishlist="toggleWishlist(i)"
+            @click:wishlist="$router.push({name: 'product', params: {id: product.id}})"
```

Comment:
temporary workaround, just ignore it

Reply:
we don't want to change route on withlist button click :P

Reply:
It's fine, I restored it